packages/block-library/src/details-summary/style.scss

```diff
@@ -0,0 +1,3 @@
+.wp-block-details-summary {
+	cursor: pointer;
```

Comment:
I'm not seeing this style when testing in the frontend. Not sure why 🤔

Reply:
I tested on MacOS in Safari, Chrome, and Firefox, and I can see the pointer on the front 🤔

Reply:
It's working for me now too!

It's strange, I'd even inspected the element and could see there was no `cursor` style.

Oh well, nothing to worry about then.